src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let styleArray = sourceString.trim().split(';');
+
+  styleArray = styleArray.map(item => item.split(':'));
+
+  styleArray = styleArray.map(item => item.map(element => element.trim()));
+
+  styleArray = styleArray.filter(item => item.length !== 1);
```

Comment:
You can use const here, and use all the methods at once

Reply:
Yep, better to rewrite using chained array methods, since they all return a value.